db/migrate/20210817144508_drop_unused_columns.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
So we have to make sure this happens after the above migration, and doesn't run if the above one fails? (To make sure descriptor_fields are all migrated out before dropping the column)

Reply:
I saw why descriptor_fields is going, because the info is stored in the 'descriptors' field, but why are the other 3 fields being axed?

Reply:
Migrations will run in order of their timestamps, and won't run if the previous one fails. You'd have to go out of your way to manually run this one first.

description isn't being axed, just changed to a string column, rather than text as its only used to store the step name. (The largest copy in the database is 33 characters long)

filename and data have never been used. They were added to allow the storing of files, but the implementation has long since decayed that it wouldn't work. If we did want to store files against steps in future, we'd use the carrierwave back documents/db_files tables.